src/parser/statement.js

```diff
@@ -1134,17 +1219,14 @@ export default class StatementParser extends ExpressionParser {
     return undefined;
   }
 
-  parsePrivateClassProperty(
+  parseClassPrivateProperty(
     node: N.ClassPrivateProperty,
   ): N.ClassPrivateProperty {
     this.state.inClassProperty = true;
 
-    if (this.match(tt.eq)) {
-      this.next();
-      node.value = this.parseMaybeAssign();
-    } else {
-      node.value = null;
-    }
+    if (node.computed !== undefined) node.computed = undefined;
```

Comment:
What's this for?

Reply:
Yeah, that was a dodgy hack for some test failure because I had set `computed` elsewhere by accident. I've fixed this now.